ui-admin/src/study/participants/datarepo/DatasetDashboard.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import { getDatasetListViewPath, StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import Api, { DatasetDetails, DatasetJobHistory } from 'api/api'
+import LoadingSpinner from 'util/LoadingSpinner'
+import { Store } from 'react-notifications-component'
+import { failureNotification } from 'util/notifications'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExternalLink, faArrowLeft } from '@fortawesome/free-solid-svg-icons'
+import { instantToDefaultString } from '../../../util/timeUtils'
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  SortingState,
+  useReactTable
+} from '@tanstack/react-table'
+import { sortableTableHeader } from '../../../util/tableUtils'
+import { Link, useParams } from 'react-router-dom'
+
+const DatasetDashboard = ({ studyEnvContext }: {studyEnvContext: StudyEnvContextT}) => {
+  const { currentEnvPath } = studyEnvContext
+  const [datasetDetails, setDatasetDetails] = useState<DatasetDetails | null>(null)
+  const [datasetJobHistory, setDatasetJobHistory] = useState<DatasetJobHistory[]>([])
+  const [isLoading, setIsLoading] = useState(true)
+  const [sorting, setSorting] = React.useState<SortingState>([])
+  const datasetName = useParams().datasetName as string
+
+  const columns = useMemo<ColumnDef<DatasetJobHistory, string>[]>(() => [{
```

Comment:
You can pull this column definition out of the component, since none of the column definitions themselves are dependent on the data.  see MailingListView.tsx for an example.  This will save a useMemo call and make the definitions clearer

Reply:
Done in c30b794f7fa660fc64a29ad06b1e3583583f76a1